Packs/CiscoEmailSecurity/Integrations/CiscoEmailSecurity/CiscoEmailSecurity.yml

```diff
@@ -0,0 +1,1220 @@
+category: Email Gateway
+commonfields:
+  id: CiscoEmailSecurity
+  version: -1
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: base_url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: API Username
+  hidden: false
+  name: api_username
+  required: true
+  type: 0
+- display: API Password
+  hidden: false
+  name: api_password
+  required: true
+  type: 4
+description: Cisco Email Security is an email security gateway . It detects and blocks
+  a wide variety of email-borne threats, such as malware, spam and phishing.
+display: Cisco Email Security
+name: CiscoEmailSecurity
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Date from which the search begins. Template for date - YYYY-MM-DD
+        hh:mm:00
+      isArray: false
+      name: start_date
+      required: true
+      secret: false
+    - default: false
+      description: Date the search ends. Template for date - YYYY-MM-DD hh:mm:00
```

Comment:
Please add a note stating that the seconds are ignored, due to API limits, here and everywhere